P5-251: Updated snippets to match code (#175)
aau-hcl-p5/main: report/5.Solution/Scheduling.tex

The system only has one task, which is declared with the properties shown on \autoref{lst:MainTask}.
The properties specifies to the operating system how the task shall run and that the event from before is related to the task.
\begin{lstlisting}[language=CSharp,label={lst:MainTaskoil},caption={MainTaks task from nxt.oil}]
TASK MainTask {
    AUTOSTART = TRUE
    {
        APPMODE = appmode1;
    };
    PRIORITY = 1;
    ACTIVATION = 1;
    SCHEDULE = FULL;
    STACKSIZE = 512;
    RESOURCE = USB_Rx;
    EVENT = newMajorCycleEvent;
};
\end{lstlisting}

> The reasoning behind simply having as major cycle with no minor cycles is the execution time of the individual methods, which can be seen on \autoref{table:executionTimes}.

Reviewer: ```suggestion
 The reasoning behind having a major cycle with no minor cycles is that the execution time of the individual methods, which can be seen on \autoref{table:executionTimes1}.
```

Update\_Display     & 267      & 15000                   \\
Background\_Task	 & 0	    & 15000					   \\
\end{tabular}

> \end{table}\label{table:executionTimes}

Reviewer: ```suggestion
 \end{table}\label{table:executionTimes1}
```

The \texttt{counter} attribute specifies that the system clock is used to time when the alarm should be run, and which action it should take upon activation.
In the autostart block, it is defined that the cycle time is 1, meaning that the task will run once every 1 update of the SysTimerCnt, which happens once every 1 millisecond, thus ensuring that the task will run every millisecond.

> The remaining tasks are also running on a ALARM, with the exception of move, and will be allowed to run based on their priority with no pre-defined release time.

Reviewer: ```suggestion
 The remaining tasks are also running on an ALARM, with the exception of move, and will be allowed to run based on their priority with no pre-defined release time.
```

Move is run using an event, which is fired in the \texttt{Receive\_Data} task.

> This is done to make sure that the turret is only rotating when required, and make sure that movement is not happening based on outdated information.

Reviewer: ```suggestion
 This is done to make sure that the turret only rotates when required, and make sure that movement is not happening based on outdated information.
```
